cluster-autoscaler-1.28.yaml

```diff
@@ -1,10 +1,13 @@
 package:
-  name: cluster-autoscaler
+  name: cluster-autoscaler-1.28
   version: 1.28.0
-  epoch: 1
+  epoch: 2
```

Comment:
This bumped so that the `package.full-version` wins the virtual over the pre-existing package.